protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2411,7 +2411,7 @@ message GetPartyRequest {
 // Response from getting party by ID
 message GetPartyResponse {
   // Data for the party requested.
-  vega.Party party = 1;
+  Party party = 1;
```

Comment:
Is it ok to delete this and not have a breaking change?

Reply:
Actually, we are not going down that route. will revert it.